v7/section_prevnext/section_prevnext.plugin

```diff
@@ -0,0 +1,12 @@
+[Core]
+name = section_prevnext
+module = section_prevnext
+
+[Documentation]
+author = Alexander Krimm
+version = 0.8
+website = https://getnikola.com/
```

Comment:
Unfortunately it is not up yet, i've updated it to point to the plugin page itself.